modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/BulkRequestProcessor.java

```diff
@@ -254,6 +305,16 @@ private String extractIDFromPath(String path) throws BadRequestException {
         }
     }
 
+    private String extractIDFromV2Path(String path) throws BadRequestException {
+        String [] parts = path.split("[/]");
```

Comment:
missing new line after the method signature

Reply:
Fixed